sample/src/test/java/com/microsoft/aad/adal/sample/ExampleUnitTest.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.aad.adal.sample;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class ExampleUnitTest {
```

Comment:
Does this file need to be in the repo or did some tool generate it.  It doesn't seem like we need a sample unit test.